api/src/main/resources/org/openmrs/api/db/hibernate/Condition.hbm.xml

```diff
@@ -29,6 +29,7 @@
 				  column="onset_date" length="19" />
 		<property name="endDate" type="java.util.Date"
 				  column="end_date" length="19" />
+         <property name="dateChanged" type="java.util.Date" column="date_changed"/>
```

Comment:
You are missing the length attribute

Reply:
@wluyima i have added the length attribute